### PR TITLE
Update setup.py

### LINKED ISSUE #38: Add and release support for Django 2.2
Please merge e964416 from https://github.com/isotoma/djangocms-column and release to PyPI to add support for Django 2.2 which is the current Django LTS release.

### PR DESCRIPTION
**Pull request by jandd**
fixes #38

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 INSTALL_REQUIRES = [
     'django-cms>=3.4.5',
-    'Django>=1.11,<2.2',
+    'Django>=1.11,<2.3',
```
Comment (Aiky30) [inline setup.py]: There will be no 2.3, this should be <3.0.  https://www.djangoproject.com/weblog/2015/jun/25/roadmap/

**Comment by lggwettmann** [comment]
Awesome and thanks for the work, when can this be shipped? :)

**Comment by bjackson** [comment]
Are there any blockers to merging this PR?

**Comment by Aiky30** [comment]
Made the change to Django <3.0 here: https://github.com/divio/djangocms-column/pull/40/files

**Comment by jandd** [comment]
Please merge #40 insteade

**Pull request closed without merging**